components/TextFields/examples/TextFieldManualLayoutLegacyExample.swift

```diff
@@ -147,11 +152,10 @@ final class TextFieldManualLayoutLegacySwiftExample: UIViewController {
 
   func setupScrollView() {
     view.addSubview(scrollView)
-    scrollView.frame = view.bounds
 
     scrollView.contentSize =
       CGSize(width: scrollView.bounds.width - 2 * LayoutConstants.largeMargin,
-             height: 372.0)
+             height: 500.0)
```

Comment:
Question: Just curious about this magic number? If we're trying to get "bigger than the screen" does it make sense to check the VC's height and double/triple?

Reply:
It's not bigger than the screen; it's enough space for all the text fields' heights put together. 

I don't like it anymore than you do but we have users that do their layout this way. :)

Reply:
Got it. Thanks!